### PR TITLE
resize: Defer any kind except dcs to kubectl

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

File: pkg/deploy/resizer/resize.go [hunk 1]
```diff
@@ -21,15 +20,12 @@ type DeploymentConfigResizer struct {
...
-		c := kubectl.NewResizerClient(kc)
-		return kubectl.ResizerFor(kind, c)
-	case "DeploymentConfig":
+	if kind == "DeploymentConfig" {
 		c := NewResizerClient(osc, kc)
 		return &DeploymentConfigResizer{c}, nil
 	}
-	return nil, fmt.Errorf("no resizer has been implemented for %q", kind)
+	c := kubectl.NewResizerClient(kc)
+	return kubectl.ResizerFor(kind, c)
```
Comment (mfojtik) [inline pkg/deploy/resizer/resize.go]: @kargakis why do you need the 'c' variable here?

File: pkg/deploy/resizer/resize.go [hunk 2]
```diff
@@ -21,15 +20,12 @@ type DeploymentConfigResizer struct {
 // ResizerFor returns the appropriate Resizer client depending on the provided
 // kind of resource (Replication controllers and deploymentConfigs supported)
 func ResizerFor(kind string, osc *client.Client, kc *kclient.Client) (kubectl.Resizer, error) {
-	switch kind {
-	case "ReplicationController":
-		c := kubectl.NewResizerClient(kc)
-		return kubectl.ResizerFor(kind, c)
-	case "DeploymentConfig":
+	if kind == "DeploymentConfig" {
```
Comment (mfojtik) [inline pkg/deploy/resizer/resize.go]:
```
if != "DeploymentConfig" {
  return kubectl.ResizerFor(kind, kubectl.NewResizerClient(kc))
}
...
```

File: pkg/deploy/resizer/resize.go (hunk 1 above)
Comment (kargakis) [inline pkg/deploy/resizer/resize.go]: `ResizerFor` needs a `ResizerClient` to work. Or are you talking about the naming of the variable?

Comment (kargakis) [inline pkg/deploy/resizer/resize.go]: Well, I could pas the whole function as well if that's what you suggest.

**Comment by mfojtik** [comment]
@kargakis :+1: 

LGTM

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**